car_to_bike.md

```diff
@@ -24,11 +26,11 @@ Seattle has many bike shops that sell both new and used bicycles. If you don't c
 
 Bike Accessories
 ----------------
-Helmets are enforced in Seattle, so make sure you’ve got one. If it’s suffered any trauma (even falling off the counter), you may want to consider buying a new one since any damage can cause the helmet to be ineffective. 
+Helmets are enforced in Seattle, so make sure you’ve got one. If it’s suffered any trauma (even falling off the counter), you may want to consider buying a new one since any damage can cause the helmet to be ineffective.
```

Comment:
There are requirements for reflectors as well as helmets.  This link is to Seattle's bike rules.  Section 11.44.160 is on lamps and reflectors.
http://www.seattle.gov/transportation/bikecode.htm